README.md

```diff
@@ -139,19 +76,29 @@ Example command:
 
 Clone this repository to your desired folder:
 
+- Click on the green button on the top right corner with the text code
+- Click on SSH
+- Copy the text that appears below
+- Type "git clone" in your terminal and past the copied text next to "git clone" and press enter
 <!--
 Example commands:
 
 ```sh
   cd my-folder
   git clone git@github.com:myaccount/my-project.git
 ```
+
 --->
 
 ### Install
 
 Install this project with:
 
+- Click on the green button on the top right corner with the text code
+- Click on "Download ZIP"
+- Extract the downloaded file locally
+- Once oponed up in a code editor run `npm install` to install all packages
```

Comment:
- [ ] You may consider removing the command that is not applied in the project. there's no node package manager `npm` dep in this project, Kinldy remove that.